src/winetricks

```diff
@@ -8207,14 +8208,78 @@ load_dxvk()
 {
     # https://github.com/doitsujin/dxvk
     _W_dxvk_version="$(w_get_github_latest_release doitsujin dxvk)"
-    _W_dxvk_version="${_W_dxvk_version#v}"
-    w_linkcheck_ignore=1 w_download "https://github.com/doitsujin/dxvk/releases/download/v${_W_dxvk_version}/dxvk-${_W_dxvk_version}.tar.gz"
+    _W_dxvk_version="v${_W_dxvk_version#v}"
+    w_linkcheck_ignore=1 w_download "https://github.com/doitsujin/dxvk/releases/download/${_W_dxvk_version}/dxvk-${_W_dxvk_version}.tar.gz"
```

Comment:
Hi again @ThisNekoGuy. I think the original change was fine. I didn't see the explanation from your commit message, just the diff itself. My apologies for misunderstanding.

Please adjust the commit message though, to something like:
"dxvk: fix link to release notes"

(doesn't have to be exactly that, feel free to tweak/explain, but commit message should clearly indicate dxvk, not just a line number).

Reply:
[Like this?](https://github.com/Winetricks/winetricks/pull/2033/commits/e5f977eeafa4fc8ec32d9ab684d407af88f2c8c8)